src/css/layout/_l-primary-nav.scss

```diff
@@ -28,14 +28,13 @@
 	// Supports
 	@supports #{$supports-flex} {
 		display: flex;
-		flex-direction: column;
 		flex-flow: row wrap;
 		margin-top: 0;
 
 		// Breakpoints
 		@include mappy-bp(palm-small) {
-			flex-direction: row;
```

Comment:
Looks like this is getting moved out of alphabetical order.

Reply:
This is directly related to the [rule `declaration-block-no-shorthand-property-overrides`](https://stylelint.io/user-guide/rules/declaration-block-no-shorthand-property-overrides) configured in our stylelint that `flex-flow: initial` is a shorthand which, by its placement afterwards, was overriding the `flex-direction: row` that was previously set before. In effect, `flex-direction: row` may be having no effect here because of that override.

As `flex-flow` is [just a combination](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-flow) of `flex-direction` and `flex-wrap`, I think a fitting solution might be to just change `flex-flow: initial` to `flex-wrap: initial`. Would that be preferred?